_haikus/hacktoberfest

```diff
@@ -0,0 +1,9 @@
+---
+layout: hailus
+title: Hacktoberfest
+Author: myself
+---
+
+*Its Hacktoberfest
```

Comment:
Not sure if you would like an asterisk here, it will not change formatting.

In this case, the first word should be spelled `It's`, standing for `it is`. 

Make sure to add a single space ` ` or `<br>` at the end of each line for line breaks.